src/Symfony/Bridge/Doctrine/composer.json

```diff
@@ -31,7 +31,7 @@
         "symfony/doctrine-messenger": "^6.4|^7.0",
         "symfony/expression-language": "^6.4|^7.0",
         "symfony/form": "^6.4.6|^7.0.6",
-        "symfony/http-kernel": "^6.4|^7.0",
+        "symfony/http-kernel": "^7.1",
```

Comment:
Should we rather keep the old behavior if `NearMissValueResolverException` doesn’t exist?

Reply:
that'd be fine to me indeed

Reply:
Okay added conditions on `NearMissValueResolverException` existence